src/models/repos.js

```diff
@@ -1,27 +1,36 @@
-const con = require('../helpers/mysqlConnect')
+const { execQuery } = require('../helpers/mysqlConnect')
 
-function getRepo (url, callback) {
-  con.query('SELECT count(*) AS num FROM repositories WHERE remote_url=?', [url], (err, rows) => {
-    if (err) throw err
-    callback(rows[0].num)
-  })
+// rename to checkForRepo?
+const getRepo = async url => {
+  const [rows] = await execQuery(
+    'SELECT count(*) AS num FROM repositories WHERE remote_url=?',
+    url
+  )
+
+  return rows[0].num
 }
 
-function addRepo (url, callback) {
-  con.query(`
-    INSERT INTO repositories
-    SET remote_url=?
-  `, [url], (err, result) => {
-    if (err) throw err
-    const id = result.insertId
-    con.query('UPDATE repositories SET location=? WHERE id=?',
-      [id, id],
-      (err2) => {
-        if (err2) throw err2
-        callback(id)
-      })
-  })
-  return []
+const addRepo = async url => {
+  const [rows] = await execQuery(
+    'INSERT INTO repositories SET remote_url=?',
+    url
+  )
+  const id = rows.insertId
+
+  await execQuery('UPDATE repositories SET location=? WHERE id=?', id, id)
```

Comment:
3rd param is redundant

Reply:
I've  changed syntax back to default, think it's better not to change it. Now it looks like:  
```javascript
await execQuery('UPDATE repositories SET location=? WHERE id=?', [id, id])
```
Is it ok like that, or we don't need second `id` at all?

Reply:
It's ok.